traffic_ops/traffic_ops_golang/staticdnsentry/staticdnsentry.go

```diff
@@ -103,6 +104,13 @@ func (staticDNSEntry TOStaticDNSEntry) Validate() error {
 		addressErr = validation.Validate(staticDNSEntry.Address, validation.Required, is.IPv6)
 	case "CNAME_RECORD":
 		addressErr = validation.Validate(staticDNSEntry.Address, validation.Required, is.DNSName)
+		address := *staticDNSEntry.Address
+		if len(address) != 0 {
```

Comment:
actually, why bother doing this if addressErr is not nil? so maybe

`if addressErr == nil && len(address) != 0`

Reply:
why check anything if len(address) = 0? If len != 0, only then I want to check addressErr and for trailing period.

Reply:
updated